Resources/Prototypes/Atmospherics/reactions.yml

```
@@ -91,6 +91,79 @@
+  - 0     # vapor
+  - 0     # miasma
+  - 10    # n2o
+  - 0     # frezon
+  effects:
+  - !type:BZFormationReaction {}
+
+- type: gasReaction
+  id: HealiumProduction
+  priority: 2
+  maximumTemperature: 300.0
+  maximumTemperature: 22.0
```

yamllint (1.35.1)

[error] duplication of key "maximumTemperature" in mapping

(key-duplicates)